examples/Example/src/new_api/camera/index.tsx

```diff
@@ -0,0 +1,351 @@
+import React, { useState } from 'react';
+import { Alert, StyleSheet, useWindowDimensions } from 'react-native';
+import {
+  GestureMonitor,
+  Gesture,
+  useAnimatedGesture,
+} from 'react-native-gesture-handler';
+import Animated, {
+  useSharedValue,
+  useAnimatedStyle,
+  withTiming,
+  runOnJS,
+} from 'react-native-reanimated';
+
+const filters = ['red', 'green', 'blue', 'yellow', 'orange', 'cyan'];
+const MAX_VIDEO_DURATION = 60_000;
+const CAPTURE_BUTTON_RADIUS = 50;
+const FILTER_BUTTON_RADIUS = 35;
+
+export default function Home() {
+  const filter = useSharedValue(0);
+  const filterOffset = useSharedValue(0);
+  const zoom = useSharedValue(1);
+  const [scale, setScale] = useState(1);
+  const [selectedFilter, setSelectedFilter] = useState(0);
+  const [isRecording, setIsRecording] = useState(false);
+  const [remainingTime, setRemainingTime] = useState(MAX_VIDEO_DURATION);
```

Comment:
```suggestion
  const [remainingTimeMs, setRemainingTimeMs] = useState(MAX_VIDEO_DURATION_MS);
```

Reply:
Done in ae78c347855f8ebb233e8fa59d2e0794bc4691a3